UI/Sign-in Page/sign.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="sign.css">
+    <title>Sign-in Page</title>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+            <h2>Sign In</h2>
+        </div>
+    
+    <form action="form" class="my-form">
+        <div class="input-group">
+            <label for="Email" class="labels">Email</label>
+            <input type="email" name="email" id="email" placeholder="someone@xyz.com">
+        </div>
+        <div class="input-group">
+            <label for="password" class="labels">Password</label>
+            <input type="password" name="password" id="password" placeholder="Password">
+        </div>
+        <div class="input-group">
+        <button class="btn">Sign In</button>
+        </div>  
+     </form>
+    </div>
+    <footer class="main-footer">
+        <div class="footer-container">
+            <nav class="footer-nav">
+                <ul>
+                <li><a href="#"><img src="/UI/Sign-in Page/Images/github.svg" alt="github" class="svgs"><span class="words">GitHub</span></a></li>
```

Comment:
as images folder is in the same folder as sign.html, you can change the src path to : src="./Images/github.svg"